etc/profile-a-l/gh.profile

```diff
@@ -0,0 +1,97 @@
+# Firejail profile for gh
+# Description: The GitHub CLI
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include gh.local
+# Persistent global definitions
+include globals.local
+
+blacklist ${RUNUSER}
+
+noblacklist ${HOME}/.config/git
+noblacklist ${HOME}/.config/nano
+noblacklist ${HOME}/.emacs
+noblacklist ${HOME}/.emacs.d
+noblacklist ${HOME}/.git-credential-cache
+noblacklist ${HOME}/.git-credentials
+noblacklist ${HOME}/.gitconfig
+noblacklist ${HOME}/.gnupg
+noblacklist ${HOME}/.nanorc
+noblacklist ${HOME}/.vim
+noblacklist ${HOME}/.viminfo
+
+# Allow environment variables with auth tokens (removed by disable-common.inc)
+ignore rmenv GH_ENTERPRISE_TOKEN
+ignore rmenv GH_TOKEN
+ignore rmenv GITHUB_ENTERPRISE_TOKEN
+ignore rmenv GITHUB_TOKEN
+
+# Allow ssh (blacklisted by disable-common.inc)
+include allow-ssh.inc
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+#include disable-X11.inc # x11 none
+include disable-xdg.inc
+
+mkdir ${HOME}/.config/gh
+whitelist ${HOME}/.config/gh
+whitelist ${HOME}/.config/git
+whitelist ${HOME}/.config/nano
+whitelist ${HOME}/.emacs
+whitelist ${HOME}/.emacs.d
+whitelist ${HOME}/.git-credential-cache
+whitelist ${HOME}/.git-credentials
+whitelist ${HOME}/.gitconfig
+whitelist ${HOME}/.gnupg
+whitelist ${HOME}/.nanorc
+whitelist ${HOME}/.vim
+whitelist ${HOME}/.viminfo
+include whitelist-common.inc
```

Comment:
```suggestion
```

As mentioned in other recent PRs, this tool is intended for development and so
whitelisting in `${HOME}` cannot be done by default without breaking most
setups.